### DISCUSSION TITLE
text transition between page

### DISCUSSION THREAD

**Discussion by Vedant-LC** [shuding/next-view-transitions#20] [Q&A]
# how did you transition text between page?
- use of className?
- use of other dependecies

### if you used className can you explain me how does this work and how can i implement by modifying it

**Comment by itsMathew** [comment]
@shuding could you please answer this?

**Comment by roman910dev** [comment]
It is achieved using the [`view-transition-name`](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/view-transition-name) CSS property.

In the demo: https://github.com/shuding/next-view-transitions/blob/b6890fb821df13b92ae79cbaf560660752a27c7e/example/app/style.css#L131
